src/System.Linq.Queryable/tests/System.Linq.Queryable.Tests.csproj

```diff
@@ -53,9 +53,11 @@
     <Compile Include="SequenceEqualTests.cs" />
     <Compile Include="SingleOrDefaultTests.cs" />
     <Compile Include="SingleTests.cs" />
+    <Compile Include="SkipLastTests.netcoreapp1.1.cs" />
```

Comment:
Likewise, this is being included for all versions, but should be netcoreapp1.1 only.